include/ofi_hmem.h

```diff
@@ -105,6 +105,15 @@ int cuda_hmem_cleanup(void);
 bool cuda_is_addr_valid(const void *addr);
 int cuda_host_register(void *ptr, size_t size);
 int cuda_host_unregister(void *ptr);
+int ofi_hmem_cuda_dev_register(struct fi_mr_attr *mr_attr, uint64_t *handle);
+int ofi_hmem_cuda_dev_deregister(uint64_t handle);
```

Comment:
Rename to just cuda_dev_xxx, and drop the ofi_hmem_ prefix.  Though are these abstractions even needed?  I'm not finding where they are used.

Reply:
They are going to be used in https://github.com/ofiwg/libfabric/pull/6219. As utility function I think it is better to have the ofi prefix? like ofi_cudaMemcpy?

Reply:
We went with just the cuda prefix for the cuda calls, like the ones above this, to avoid prefix overload.

Reply:
Fixed in new revision.